samples/subsys/pm/device_pm/README.rst

```diff
@@ -0,0 +1,40 @@
+.. zephyr:code-sample:: device-pm
+   :name: Device Power Management
+   :relevant-api: device-runtime-pm-apis
```

Comment:
This was actually ok! Sorry that this is confusing :)
This, is for augmenting the beginning of each API reference section with cross links to the code sample
The `:ref:`s that needed to be changed are because linking to an actual Doxygen group is actually not easily possible (go figure) so you'd want to link to a manually created anchor instead.
Hope this helps!

Reply:
Oh, I see. Thanks for the clarification! I will change these references to what they were before.